terraform/datadog/concourse.tf

```diff
@@ -52,6 +52,24 @@ resource "datadog_monitor" "continuous-smoketests-failures" {
   tags = ["deployment:${var.env}", "service:${var.env}_monitors"]
 }
 
+resource "datadog_monitor" "concourse-btrfs" {
+  name                = "${format("%s concourse excess btrfs processes running", var.env)}"
+  type                = "query alert"
+  message             = "Large number of btrfs processes running. Concourse probably needs shaking."
```

Comment:
@dcarley Done, and a PR for the team manual opened: https://github.com/alphagov/paas-team-manual/pull/113